docs/usage/node-local-dns.md

```diff
@@ -45,6 +45,7 @@ It is worth noting that:
 - During the reconfiguration of the node-local-dns there might be a short disruption in terms of domain name resolution depending on the setup. Usually, DNS requests are repeated for some time as UDP is an unreliable protocol, but that strictly depends on the application/way the domain name resolution happens. It is recommended to let the shoot be reconciled during the next maintenance period. 
 - If a short DNS outage is not a big issue, you can [trigger reconciliation](./shoot_operations.md#immediate-reconciliation) directly after setting the annotation.
 - Switching node-local-dns off by removing the annotation can be a rather destructive operation that will result in pods without a working DNS configuration.
+- Enabling or disabling node-local-dns triggers a rollout of all shoot worker nodes, see also [this document](shoot_updates.md#rolling-update-triggers).
```

Comment:
As the node-local-dns annotations were dropped in https://github.com/gardener/gardener/pull/8364, could you please also change the documentation accordingly while you are already at it?

Lines 44 and 47 mention the annotation. Both lines should be removed in favour of the newly added line.